static/app/app/list/views.js

```diff
@@ -33,10 +35,24 @@ var LunchMoveView = Marionette.ItemView.extend({
         channel.trigger('edit');
         return false;
     },
+    activateTooltips: function() {
+        $(function () {
```

Comment:
This shouldn't need to be wrapped as a [callback on document load](http://api.jquery.com/jquery/#jQuery3) as the document should already be loaded by this point.